prover/src/default_values.rs

```diff
@@ -18,12 +18,14 @@ pub const UBV_PK: &str = "ubv.pk";
 pub const UBV_VK: &str = "ubv.vk";
 pub const UBV_GATE_CONFIG: &str = "ubv.specs";
 pub const UBV_PROTOCOL: &str = "ubv.protocol";
+pub const UBV_PROOF: &str = "ubv.proof";
```

Comment:
Only used as the base for the `full prove` intermediate files, AFAICT?

Reply:
yes

Reply:
```suggestion
pub const UBV_PROOF_BASE: &str = "ubv.proof";
```

Reply:
changed